next.config.js

```diff
@@ -30,3 +31,40 @@ const nextConfig = {
 }
 
 module.exports = nextTranslate(nextConfig)
+
+// Injected content via Sentry wizard below
+
+const { withSentryConfig } = require('@sentry/nextjs')
+
+module.exports = withSentryConfig(
```

Comment:
I suggest to use more simplified version for the beginning: https://github.com/optriment/optrispace-frontend/blob/master/next.config.js